_quarto.yml

```diff
@@ -3,7 +3,7 @@ project:
   render:
     - "index.qmd"
     - "slides/*qmd"
-    - "!archive/"
+    - "archive/2022-07-RStudio-conf/*qmd"
```

Comment:
We will manually update this file in this way when we have new archived versions.